### PR TITLE
rocksdb: basic tuning

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang PTAL

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -71,6 +71,11 @@ fn build_raftkv(matches: &Matches,
     block_base_opts.set_block_size(64 * 1024);
     opts.set_block_based_table_factory(&block_base_opts);
     opts.compression(DBCompressionType::DBNo);
+    opts.set_write_buffer_size(64 * 1024 * 1024);
+    opts.set_max_write_buffer_number(5);
+    opts.set_min_write_buffer_number_to_merge(2);
+    opts.set_max_background_compactions(3);
+    opts.set_max_bytes_for_level_base(512 * 1024 * 1024);
     opts.set_target_file_size_base(64 * 1024 * 1024);
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: I think we should set LRU cache and allow OS cache too.

Comment (ngaut) [inline src/bin/tikv-server.rs]: +1

**Comment by ngaut** [comment]
LGTM， LRU cache could be in next PR.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**